README.md

```diff
@@ -0,0 +1,71 @@
+identify
+========
+
+File identification library for Python.
+
+Given a file (or some information about a file), return a set of standardized
+tags identifying what the file is.
+
+
+## Usage
+### With a file on disk
+
+If you have an actual file on disk, you can get the most information possible
+(a superset all other methods):
+
+```python
+>>> identify.tags_from_path('/path/to/file.py')
+{'file', 'text', 'python', 'non-executable'}
+>>> identify.tags_from_path('/path/to/file-with-shebang')
+{'file', 'text', 'shell', 'bash', 'executable'}
+>>> identify.tags_from_path('/bin/bash')
+{'file', 'binary', 'executable'}
+>>> identify.tags_from_path('/path/to/directory')
+{'directory'}
+>>> identify.tags_from_path('/path/to/symlink')
+{'symlink'}
+```
+
+When using a file on disk, the checks performed are:
+
+* File type (file, symlink, directory)
+* Mode (is it executable?)
+* File name (mostly based on extension)
+* If executable, the shebang is read and the interpreter interpreted
+
+
+### If you only have the filename
+
+```python
+>>> identify.tags_from_filename('file.py')
+{'text', 'python'}
```

Comment:
conventionally `.py` files are "text", but it's possible that it isn't.  Shouldn't the `text` tag only be applied to on-disk things?